### PR TITLE
Adding a new Tank and high/low level monitoring abstract type

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**
Adding Entity Type: TK_LVLHL_SWTM
Adding Abstract Type: LVLHL
@kevin-hereworks

### CODE REVIEW

**Comment by Ciaran-Oflannagain** [comment]
@shambergoldstein - just adding two additional tanks using the submitted abstract type LVLHL. Thanks, Ciaran

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -7044,6 +7044,13 @@ LM:
   - level_status
   - percentage_sensor
 
+LVLHL:
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:7047]: I would suggest changing the name of this abstract type to something like LHLM (level high low monitoring) to be more consistent with the existing level monitoring abstract type, LM, and the general way abstract types are structured (control/monitoring tag on the end).

File: ontology/yaml/resources/HVAC/entity_types/TANK.yaml [hunk 2]
```diff
@@ -16,6 +16,30 @@
 ### Canonical Types  ###
 ########################
 
+TK_SS_LVLHL:
+  guid: "f6beac97-8d0b-4668-a7a4-0352b7205ba2"
+  description: "Tank with high and low water level monitoring."
+  is_canonical: true
+  implements:
+  - SS
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/TANK.yaml:24]: What is the SS referring to here? (What is the run command and status for?)

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 1 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: @shambergoldstein - thank you, that makes a lot more sense. Have changed from LVLHL to LHLM.

File: ontology/yaml/resources/HVAC/entity_types/TANK.yaml (hunk 2 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/TANK.yaml]: Apologies no need for SS here. Thanks, Ciaran

**Review by shambergoldstein — APPROVED** [review]

**Pull request merged**